include/envoy/http/access_log.h

```diff
@@ -10,29 +10,31 @@
 namespace Http {
 namespace AccessLog {
 
-enum class FailureReason {
+enum ResponseFlag {
   // No failure.
-  None,
+  None = 0x0,
```

Comment:
this way caller can check if no flags are set otherwise we'd need to have separate method isAnyResponseFlagSet or something along these lines.
Open to remove None if there are better ways of handling situation above.